test/setUnion_test.js

```diff
@@ -0,0 +1,23 @@
+import { expect } from "chai"
+import setUnion from "../src/setUnion"
+
+describe('setUnion', function() {
+
+	it('should return union of two sets', function() {
+		const a = [1, 2, 3, 4]
+		const b = [2, 4, 6, 8]
+		expect(setUnion(a, b)).to.deep.equal([1, 2, 3, 4, 6, 8]) 
+	})
+
+	it('should return b if a is null', function() {
+		const b = null
+		const a = [2, 4, 6, 8]
+		expect(setUnion(a, b)).to.deep.equal([2, 4, 6, 8]) 
+	})
```

Comment:
If this tests for if either are null I would make the it statement more clear. If it does not I would add another test for if a is null.